scripts/dashboard/validate_dashboards_format.py

```diff
@@ -0,0 +1,153 @@
+'''This script will be triggered by a github action outlined in ~/.github/workflows/validate_dashboards_formar.yml.
+It will be used to validate all new json and metadata files pertaining to integration sample dashboards.'''
+
+import json
+import os
+import sys
+import yaml
+
+class MetadataFormattingError(Exception):
+  '''Raised when data in metadata file does not have expected format'''
+  pass
+
+class JsonFormattingError(Exception):
+  '''Raised when data in json file or file name does not have expected format'''
+  pass
+
+class MetadataInJsonMisMatchError(Exception):
+  '''Raised when data in metadata file does not have matching data in a json file'''
+  pass
+
+class JsonInMetadataMisMatchError(Exception):
+  '''Raised when data in json file does not have matching data in a metadata file'''
+  pass
+
+def mapPaths(paths):
+  '''Takes in list of paths and returns dictionary of metadata and dashboard json data, 
+  organized by directories'''
+  map = {}
+
+  for path in paths:
+    pathParts = path.split('/')
+
+    # filter out non-dashboards files
```

Comment:
Can we also filter out anything beyond depth of 1 below `dashboards/` ?
For example, there is a `deprecated/` folder under MongoDB. It would be fine, and probably preferable to skip the validation for those files.

so
```python
if not 'dashboards' == pathParts[1] or len(pathParts) > 4:
  continue
```

Reply:
done.